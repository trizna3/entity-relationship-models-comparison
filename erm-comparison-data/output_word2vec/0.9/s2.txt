typy rakiet-typ rakiet
stihacie modely-stihacky
dopravne modely-dopravne lietadla
lietadla-lietadla
modely lietadiel-modely
vojaci-vojaci
role vojakov-role
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-typ rakiet-naklad
