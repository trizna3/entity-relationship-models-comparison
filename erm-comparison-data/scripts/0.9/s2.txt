entity set "lietadla" {
	"seriove cislo"
	"meno"
};

entity set "vojaci" {
	"meno"
	"zivot"
	"maximalny zivot"
	"obratnost"
	"inteligencia"
};

entity set "modely" {
	"nazov modelu"
};

entity set "role" {
	"meno"
	"modifikator maximalneho zivota"
	"modifikator zivota"
	"modifikator sily"
	"modifikator obratnosti"
	"modifikator inteligencie"
};

entity set "stihacky" {
};

entity set "dopravne lietadla" {
	"kapacita osob"
	"maximalny objem"
	"material hmotnost"
};

entity set "naklad" {
	"typ rakiet"
	"pocet"
};

association between 
{
	"lietadla" MANY
	"modely" ONE
}{
};

association "prideleny k" between 
{
	"lietadla" ONE
	"vojaci" ONE
	"role" ONE
}{
};

association "nesie" between 
{
	"lietadla" ONE
	"naklad" MANY
}{
};

association between 
{
	"stihacky" ONE
	"naklad" MANY
}{
};

generalization from "stihacky" to "modely";
generalization from "dopravne lietadla" to "modely";