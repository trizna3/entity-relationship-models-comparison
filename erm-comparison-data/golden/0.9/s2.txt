modely lietadiel-modely
stihacie modely-stihacky
typy rakiet-typ rakiet
dopravne modely-dopravne lietadla
lietadla-lietadla
vojaci-vojaci
role vojakov-role
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-typ rakiet-naklad